ecc/blocks/event-agenda-component/event-agenda-component.css

```diff
@@ -1,3 +1,19 @@
+.event-agenda-component {
+  display: flex;
+  flex-direction: column;
+  gap: 59px;
+}
+
+#agenda {
```

Comment:
I would be careful with the CSS rule here: Different from Lit components, block CSS files are imported to the same scope and will override each other. That's why you see all of the block CSS all starting with .block-name.